dataplane/linux/xdp_state.go

```diff
@@ -426,31 +426,31 @@ func (s *xdpIPState) getBpfIPFamily() bpf.IPFamily {
 	return bpf.IPFamilyUnknown
 }
 
-func (s *xdpIPState) newXDPResyncState(common *xdpStateCommon, isSource ipsetsSource) (*xdpResyncState, error) {
-	xdpIfaces, err := common.bpfLib.GetXDPIfaces()
+// newXDPResyncState creates the xdpResyncState object which represents the
+func (s *xdpIPState) newXDPResyncState(bpfLib bpf.BPFDataplane, isSource ipsetsSource, programTag string, xpdModes []bpf.XDPMode) (*xdpResyncState, error) {
```

Comment:
Agree that `ipsSource` is probably a better name, please update here too to be consistent.